src/andromede/study/data.py

```diff
@@ -101,6 +104,18 @@ def check_requirement(self, time: bool, scenario: bool) -> bool:
         return scenario
 
 
+def load_ts_from_txt(
+    timeseries_name: Optional[str], path_to_file: Optional[Path]
+) -> pd.DataFrame:
+    if path_to_file is not None and timeseries_name is not None:
+        timeseries_with_extension = timeseries_name + ".txt"
```

Comment:
Bit strange. Why add the prefix here ?

I understand that we don't want ".txt" in the yaml files, but this adds implicit knowledge (the file XXX.txt must be referenced as XXX in the yaml).

Reply:
We want to hand different extensions and that the final user should only put timeseries file name in the .yaml file